src/Balbasaur.java

```diff
@@ -0,0 +1,37 @@
+public class Balbasaur extends Pokemon {
```

Comment:
Je hebt deze klasse "Balbasaur" genoemd. De opdracht was om een "GrassPokemon" klasse te maken. Daarvan zou balbasaur dan een specifieke implementatie kunnen zijn.
Dus dan krijg je iets als: "GrassPokemon balbasaur = new GrassPokemon(...)", maar kun je ook "GrassPokemon bellSprout = new GrassPokemon(...)".

Alternatief kun je ook: "class Balbasaur extend GrassPokemon" doen, want er is niet maar 1 balbasaur in de "lore" van pokemon.